CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Contributing to Atom
```

Comment:
~Atom~ Orbitdeterminator

Reply:
It was resolved in commit [#89368d9](https://github.com/aerospaceresearch/orbitdeterminator/pull/159/commits/89368d9fcc56291b922068a9e832048871a77b07)